### PR TITLE
feat: add version.schema.json generating job to goreleaser orb

### PR DESCRIPTION
**Pull request by zepatrik**

### CODE REVIEW

File: src/orbs/goreleaser.yml [hunk 1]
```diff
@@ -112,6 +112,16 @@ jobs:
...
 
+  render-version-schema:
+    executor: default
+    working_directory: /home/ory
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run: |
+          curl -sL https://raw.githubusercontent.com/ory/ci/master/src/scripts/releaser/render-version-schema.sh | bash > $BASH_ENV
+          source $BASH_ENV
```
Comment (aeneasr) [inline src/orbs/goreleaser.yml]: Why are you sourcing the bash script? It does not include any env vars. What you want is this:

```
      -
        run:
          command: "bash <(curl -s https://raw.githubusercontent.com/ory/ci/master/src/scripts/releaser/campaign-send.sh)"
```

**Review by aeneasr — CHANGES_REQUESTED** [review]

File: src/orbs/goreleaser.yml (hunk 1 above)
Comment (zepatrik) [reply src/orbs/goreleaser.yml]: copy/paste error :sweat_smile:

File: src/scripts/releaser/render-version-schema.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -Eeuox pipefail
+
+bash <(curl https://raw.githubusercontent.com/ory/cli/master/install.sh) -b $GOPATH/bin
+
+ory dev schema render-version $CIRCLE_TAG
+git commit -A -m "autogen: update version.schema.json to include new release"
```
Comment (aeneasr) [inline src/scripts/releaser/render-version-schema.sh]:
```suggestion
git commit -A -m "autogen: add $CIRCLE_TAG to version.schema.json"
```

**Comment by aeneasr** [comment]
Available as ory/goreleaser@0.1.14

**Pull request merged**